x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -1,80 +1,146 @@
 [[configuring-stack-security]]
-== Configure security for the Elastic Stack
-++++
-<titleabbrev>Configuring security</titleabbrev>
-++++
+== Start the Elastic Stack with security enabled
 
-Security needs vary depending on whether you're developing locally on your
-laptop or securing all communications in a production environment. Because
-security needs vary, the following scenarios provide options for configuring
-the Elastic Stack.
+beta::[This functionality is in alpha and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Alpha features are not subject to the support SLA of official GA features.]
 
-TIP: Each subsequent scenario builds on the previous one so that you can add
-additional security by building on the existing layer.
-
-These scenarios don't cover every situation, but provide a framework for
-securing {es} and the Elastic Stack based on typical use cases.
-
-image::images/elastic-security-overview.png[Elastic Security layers]
+Before you start {es} for the first time, run the `elasticsearch-security-config`
+tool. This tool generates certificates and keys for the transport and HTTP
+layers of your, and writes the TLS configuration settings in `elasticsearch.yml`.
 
 [discrete]
-[[security-minimal-overview]]
-=== Minimal security ({es} Development)
-
-If you want to set up {es} on your laptop and start developing, this scenario
-is for you. This configuration prevents unauthorized access to your local
-cluster by setting up passwords for the built-in users. You also configure
-password authentication for {kib}.
-
-// tag::minimal-security-note[]
-IMPORTANT: The minimal security scenario is not sufficient for
-<<dev-vs-prod-mode,production mode>> clusters. If your cluster has multiple
-nodes, you must enable minimal security and then
-<<security-basic-setup,configure Transport Layer Security (TLS)>> between
-nodes.
+=== Prerequisites
 
-// end::minimal-security-note[]
-
-<<security-minimal-setup,Set up minimal security>>
+Download the `elasticsearch 8.0.0-alpha1` package distribution for your
+environment.
 
 [discrete]
-[[security-basic-overview]]
-=== Basic security ({es} Production)
-
-This scenario builds on the minimal security requirements by adding transport
-Layer Security (TLS) for communication between nodes. This additional layer
-requires that nodes verify security certificates, which prevents unauthorized
-nodes from joining your {es} cluster.
-
-Your external HTTP traffic between {es} and {kib} won't be encrypted, but
-internode communication will be secured.
-
-<<security-basic-setup,Set up basic security>>
+=== Start {es} with security enabled
+
+. Before starting {es} for the first time, run the
```

Comment:
Would it make sense for us to add a paragraph here saying that this is going to be taken care automatically in the future the first time you start your node and that this is early preview showcasing the direction we're taking ? Or is this implied by it being alpha1 ?

Reply:
I feel like it's implied since this is an alpha release. IMO, the docs should reflect the current state of the release. We can create a new PR for alpha2 and update the docs to match the workflow and capabilities.